extras/docker/qns/run_endpoint.sh

```diff
@@ -12,9 +12,9 @@ set -e
 QUICHE_DIR=/quiche
 WWW_DIR=/www
 DOWNLOAD_DIR=/downloads
-QUICHE_CLIENT=client
-QUICHE_SERVER=server
-QUICHE_CLIENT_OPT="--no-verify"
+QUICHE_CLIENT=quiche-client
+QUICHE_SERVER=quiche-server
+QUICHE_CLIENT_OPT="--no-verify --dump-responses ${DOWNLOAD_DIR}"
 QUICHE_SERVER_OPT="--no-retry"
```

Comment:
```suggestion
QUICHE_SERVER_OPT="--max-streams-bidi 3000 --max-streams-uni 3000 --no-retry --cert examples/cert.crt --key examples/cert.key"
```

Reply:
We shouldn't just increase the stream limit for the purpose of passing the test. If there is a bug in the implementation we should fix it.

Reply:
Oh. Probably leftover from testing. I will remove.